### ISSUE TITLE
Formulierung allgemeiner halten? Beispiele für aerosolübertragene Krankheiten geben?

### ISSUE THREAD

**Issue by MacLemon** [c3wien/Poke-That-Gmeind#116]
> Wir möchten die Sicherheit unserer Mitmenschen vor SARS-COV-2 und anderen durch die Luft übertragenen Viren erhöhen.
Evt. Viren durch Krankheiten ersetzen?

Es gibt schließlich auch Bakterielle Infektionen die so verbreitet werden. (zB Diverse -Kokken Infektionen)

Ich fände es auch gut einige Beispiele was das so sein könnte aufzuführen, da es die Problematik greifbarer macht.
Zu viele Menschen sind ja leider der irrigen Ansicht, daß SARS-CoV-2 „vorbei“ ist und fühlen sich somit nicht persönlich betroffen.

Beispiele für andere Krankheiten die ebenso durch Aerosole übertragen werden und die mit Luftfiltern eingedämmt werden können könnten den „Call to action“ näher legen.

**Comment by cambid** [comment]
Good Idea.
@pascoda: Something for you?

**Comment by eest9** [comment]
Ja, im FAQ kann man IMHO tatsächlich etwas präziser sein. Auf der Startseite würde ich den aktuellen Text lassen, da der Text ansonsten zu lange und vor allem komplex wird. (ich hab mal angenommen, dass sich der Vorschlag ohnehin nur auf die FAQ bezieht)

**Comment by eest9** [comment]
Wobei eventuell Krankheitserreger generell ein guter Begriff wäre/ist der auch für die Startseite geeignet sein könnte.

**Comment by pascoda** [comment]
[memo](https://flexikon.doccheck.com/de/Aerogene_Infektion), und [quelle](https://sci-hub.wf/doi:10.1001/jama.2020.4756)
